docs/building.md

```diff
@@ -23,3 +23,39 @@ ctest -LE performance
 # run performance tests
 ctest -L performance
 ```
+
+## LANL Darwin - Power 9
+
+The following works on the Darwin Power 9 nodes. We use Spack to install `hdf5` with the `xl`
+compiler. Setting up Spack is beyond the scop of this documentation.
+
+```bash
+# HDF5 is optional, but recommended
+spack install hdf5%xl
```

Comment:
I had trouble getting hdf5 installed I had to use:

```
spack install hdf5@1.10.6%xl@16.1 ^openmpi@4.0.3%xl@16.1 arch=rhel7-ppc64le
```

Reply:
My experience with HDF5 is that it is often easier to just build it by hand if I can't get it to behave with Spack. Spack is preferred of course. However, building HDF5 is easy and fairly foolproof.